cpu/efm32/Makefile.dep

```diff
@@ -13,4 +13,9 @@ endif
 # include CPU family module
 USEMODULE += cpu_$(CPU_FAM)
 
+ifneq (,$(filter periph_usbdev,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_usbdev
```

Comment:
```suggestion
```

I'd say features are already evaluated at this point? It's too late to require further features.

I'd say a `FEATURE_REQUIRED += periph_usbdev` in some Makefile already resulted into `periph_usbdev` being part of `USEMODULE`.

Reply:
You are right.